styleflex.css

```diff
@@ -0,0 +1,127 @@
+html {
+    width: 100%; 
+    height: 100%;
+    background-color: rgb(183, 224, 250);
+    font-size: 14px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.container {
+    background-color: white;
+    width: 1050px;
+    height: 680px;
```

Comment:
It's not recommended to set height on the container. Instead, we can just let content inside the container to control its height. The benefit is once we want to update on the contents, we can just do it without worry about the container height.